sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastHashJoinExec.scala

```diff
@@ -46,14 +46,26 @@ case class BroadcastHashJoinExec(
     buildSide: BuildSide,
     condition: Option[Expression],
     left: SparkPlan,
-    right: SparkPlan)
+    right: SparkPlan,
+    isNullAwareAntiJoin: Boolean = false)
   extends HashJoin with CodegenSupport {
 
+  if (isNullAwareAntiJoin) {
+    // TODO support multi column NULL-aware anti join in future.
```

Comment:
Perhaps this comment block can be moved to the ExtractSingleColumnNullAwareAntiJoin ? To explain why we only specialize for single columns.

Reply:
done